src/components/Tooltip/Tooltip.css

```diff
@@ -135,6 +135,10 @@
   flex: 0 0 100%;
 }
 
+.high-z-index {
```

Comment:
maybe rename it to "z-index-1001"? just to be more clear, i think we may have several z-index classes to handle layers

Reply:
Done!